saleor/graphql/file_upload/views.py

```diff
@@ -1,58 +1,177 @@
-                variables = operations.get('variables')
-                for file_key in files_map:
-                    # file key is which file it is in the form-data
-                    file_instances = files_map[file_key]
-                    for file_instance in file_instances:
-                        test = obj_set(
-                            operations, file_instance, file_key, False)
-                query = operations.get('query')
-                variables = operations.get('variables')
-            except Exception as e:
-                raise e
+    def handle_query(self, request: HttpRequest, *args, **kwargs):
+        if request.method == 'GET':
+            return render_to_response('graphql/playground.html')
+        if request.method != 'POST':
+            return HttpResponseNotAllowed(
+                ['POST'],
+                content_type='application/json',
+                content=json.dumps({
+                    'errors': [
+                        self.format_error(
+                            'GraphQL only supports POST requests.')]}))
+        try:
+            data = self.parse_body(request)
+        except ValueError as e:
+            return HttpResponseBadRequest(
+                content_type='application/json',
+                content=json.dumps({
+                    'errors': [self.format_error('Unable to parse query.')]}))
+        if isinstance(data, list):
+            responses = [self.get_response(request, entry) for entry in data]
+            result = [response for response, code in responses]
+            status_code = max((code for response, code in responses),
+                              default=200)
         else:
-            query, variables, operation_name, id = super(
-                FileUploadGraphQLView,
-                FileUploadGraphQLView).get_graphql_params(request, data)
-        return query, variables, operation_name, id
+            result, status_code = self.get_response(request, data)
+        return HttpResponse(
+            status=status_code,
+            content=json.dumps(result),
+            content_type='application/json')
+
+    def parse_body(self, request):
+        content_type = request.content_type
+        if content_type == 'application/graphql':
+            return {'query': request.body.decode('utf-8')}
+        elif content_type == 'application/json':
+            body = request.body.decode('utf-8')
+            return json.loads(body)
+        elif content_type in ['application/x-www-form-urlencoded',
+                              'multipart/form-data']:
+            return request.POST
+        return {}
+
+    def get_response(self, request: HttpRequest, data):
+        query, variables, operation_name = self.get_graphql_params(
+            request, data)
+        execution_result = self.execute_graphql_request(
+            request, data, query, variables, operation_name)
+        status_code = 200
+        if execution_result:
```

Comment:
Is this condition ever False?

Reply:
Not sure, that's the original logic from Django Graphene.